### DISCUSSION TITLE
Hide fields on `BelongsToMany` or `MorphToMany` relatable view

### DISCUSSION THREAD

**Discussion by kichetof** [laravel/nova-issues#4747] [Ideas & Feature Requests]
Hello,

It would be very helpful to hide some fields on `BelongsToMany` or `MorphToMany` when you have a lot of pivot fields.
As both resources *ToMany and PivotFields are merged, pivot fields are stacked at the end and you always need to scroll to see these fields.

Adding a method like `hideFromRelatable` could allow us to hide some fields in relatable view.

What do you think?
Thanks for your help! :)

**Comment by puzzledmonkey** [comment]
I second this request!
